client/src/components/Tags/StatelessTags.test.js

```diff
@@ -7,8 +7,9 @@ describe("Tags/StatelessTags.vue", () => {
     const localVue = createLocalVue();
     localVue.filter("localize", (value) => _l(value));
 
-    let testTags = ["abc", "def", "ghi"];
-    let wrapper, emitted;
+    const testTags = ["abc", "def", "ghi"];
+    let wrapper;
+    let emitted;
```

Comment:
is there a reason why?

Reply:
There's still a bit of a mix in the codebase, but the vast majority adheres to https://eslint.org/docs/rules/one-var.  I probably need to update the rule to enforce let/const as well.

Reply:
but still, why not together?

Reply:
Here's a good explanation I mostly agree with.  https://github.com/airbnb/javascript#variables.